lib/Scheduler/enumerator.cpp

```diff
@@ -1111,10 +1153,14 @@ bool Enumerator::ProbeBranch_(SchedInstruction *inst, EnumTreeNode *&newNode,
   if (prune_.nodeSup) {
     if (inst != NULL)
       if (crntNode_->WasSprirNodeExmnd(inst)) {
-#ifdef IS_DEBUG_INFSBLTY_TESTS
         stats::nodeSuperiorityInfeasibilityHits++;
+#ifdef IS_DEBUG_INFSBLTY_TESTS
+        // stats::nodeSuperiorityInfeasibilityHits++;
 #endif
```

Comment:
Why was this moved outside the `#ifdef`? If it should be, then delete the `#ifdef`.